webclient/components/ptinfo-single/1time-Mrow-mField/diagnosis/db/server-side/structure-gen-on-2020-09-15.sql

```diff
@@ -14,8 +14,8 @@ CREATE TABLE `assignedDiagnosis` (
   `patientUUID` char(36) NOT NULL,
```

Comment:
line no: 14. -> it should be 'ptUuid' instead of `patientUUID`

Reply:
rename field